server/src/test/resources/application.conf

```diff
@@ -0,0 +1,1189 @@
+scassandra.binary.port=8042
```

Comment:
This file must have been pulled over in the merge commit, I don't think it needs to be here does it?  Since the defaults defined in `server/src/main/resources/application.conf` are the same.